react/Text/index.spec.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Text, MainTitle, Title, SubTitle, Bold, Uppercase, Caption, ErrorMessage } from './index'
+
+describe('Text/*', () => {
+  it('forwards ref to the HTML tag', () => {
+    const components = [ Text, MainTitle, Title, SubTitle, Bold, Uppercase, Caption, ErrorMessage ]
```

Comment:
Nit: Putting the for loop just outside the `it` would allow for more precise test failures.

```
const components = [ Text, MainTitle, Title, SubTitle, Bold, Uppercase, Caption, ErrorMessage ]

for (const component of components) {
  test('${component.name} forwards ref to the HTML tag', () => {
    ...
  })
}
```